uberduck_ml_dev/models/gradtts.py

```diff
@@ -0,0 +1,1152 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/models.gradtts.ipynb (unless otherwise specified).
+
+__all__ = ['BaseModule', 'Mish', 'Upsample', 'Downsample', 'Rezero', 'Block', 'ResnetBlock', 'LinearAttention',
+           'Residual', 'SinusoidalPosEmb', 'GradLogPEstimator2d', 'get_noise', 'Diffusion', 'sequence_mask',
+           'fix_len_compatibility', 'convert_pad_shape', 'generate_path', 'duration_loss', 'LayerNorm', 'ConvReluNorm',
+           'DurationPredictor', 'MultiHeadAttention', 'FFN', 'Encoder', 'TextEncoder', 'GradTTS', 'DEFAULTS']
+
+# Cell
```

Comment:
No functional changes in this file besides changing the input parameters of GradTTS to be hparams rather than a list of hyperparameters